sspl_test/rabbitmq/rabbitmq_egress_processor.py

```diff
@@ -213,6 +215,9 @@ def _read_config(self):
             self._site_id = self._conf_reader._get_value_with_default(
                 self.SYSTEM_INFORMATION, self.SITE_ID, '')
 
+            # Decrypt RabbitMQ Password
+            decryption_key = encryptor.gen_key(self._cluster_id, ServiceTypes.RABBITMQ.value)

```

Comment:
Can you please check where are we reading self._cluster_id from?

Reply:
line 212, from common config only, which looks fine to me.